app/dashboard/templates/bounty/fulfill.html

```diff
@@ -83,6 +83,15 @@ <h4 class="text-center mb-4">{% trans "Submit Work" %}</h4>
                     />
                   </div>
 
+                  <div class="mt-2">
+                    <label class="form__label" for="projectId">{% trans "Select the submitting project" %}</label>
+                    <select name='projectId' id='projectId' class="form__input w-100 js-select2 font-body" required>
+                      {% for project in prize_projects %}
+                      <option value="{{ project.id }}">{{ project.id }} - {{ project.name }}</option>
+                      {% endfor %}
+                    </select>
+                  </div>
```

Comment:
^ why this there for every bounty 
Wrap it in a if check to check if there are prize projects please

Reply:
done :)